README.md

```diff
@@ -195,15 +195,15 @@ This can be tedious, but hopefully doesn't come up much. If it does, and you wan
 
 Also, Lua patterns are case sensitive, so the `<pattern>` of `no` will *not* match `NOOOOO` or `No, stop`, but *will* match `That's enough!`.
 
-## Sound files
+## Sound Files
 
 To play the sound files, this plugin takes advantage of the built-in HexChat `/splay <soundfile>` command, triggering it with the given sound file every time a message with `<pattern>` is received.
 
 HexChat supports [different sound file formats on different platforms](./REAMDE.md#wav-is-the-only-sound-fileformat-playable-on-windows), so test out the sound file with `/splay` before using it in a configuration, to make sure it can be played.
 
 # Limitations
 
-## Available message types
+## Available Message Types

```

Comment:
Heading title might not fit with rest of section

Reply:
## Only Channel Messages Are Matched

## Limited Message Types

Do either of these sound better?

Reply:
"Only Channel Messages Are Matched" is personally what I would go with